test/system/jsdoc-config.test.js

```diff
@@ -37,63 +36,63 @@ describe('JSDoc configuration', function () {
 
     describe('source', function () {
         it('should have an include pattern', function () {
-            expect(json.source.includePattern).to.be('.+\\.js(doc)?$');
+            expect(json.source.includePattern).to.equal('.+\\.js(doc)?$');
         });
 
         it('should have an exclude pattern', function () {
-            expect(json.source.excludePattern).to.be('(^|\\/|\\\\)_');
+            expect(json.source.excludePattern).to.equal('(^|\\/|\\\\)_');
         });
     });
 
     describe('plugins', function () {
         it('should have the markdown plugin', function () {
-            expect(_.includes(json.plugins, 'plugins/markdown')).to.be.ok();
+            expect(_.includes(json.plugins, 'plugins/markdown')).to.be.ok;
```

Comment:
`expect(json.plugins, 'Should use the markdown plugin').to.include('plugins/markdown')`